src/cascadia/TerminalApp/AppLogic.h

```diff
@@ -5,7 +5,7 @@
 
 #include "AppLogic.g.h"
 
-#include "Tab.h"
+#include "TerminalTab.h"

```

Comment:
Curious: What do we need this one for?

Reply:
hrmm you're right I don't think including `Tab` was needed in the first place